components/ui/ProgressFreemiumComponent.tsx

```diff
@@ -0,0 +1,39 @@
+import { Indicator, Root } from "@radix-ui/react-progress";
+import React, { useEffect, useState } from "react";
+
+export interface ProgressFreemiumComponentProps {
+  elapsedDays: number;
+  totalTrialDays: number;
+}
+const ProgressBarComponent: React.FC<ProgressFreemiumComponentProps> = ({
+  elapsedDays,
+  totalTrialDays,
+}) => {
+  const [progress, setProgress] = useState(0);
+  useEffect(() => {
+    setProgress(elapsedDays);
+  }, []);
+
+  return (
+    <div>
+      <Root
+        className="relative overflow-hidden bg-white rounded-full w-4/5 h-[8px]"
```

Comment:
What's the reasoning for using 4/5 instead of absolute widths? Did you check how this looks on different screen sizes?

Reply:
Couldn't you use one of the default tailwind heights instead of hard coding 8px? I think one of the classes is equivalent to 8px I think it's h-2

Reply:
The reasoning is that the current use will be in the Sidebar component, which will force the size on both mobile and computer to be limited to the sidebar width.  

Great point concerning the tailwind classes.  I used h-2 and w-40.  Checked that on mobile and computer as well.